cmds/tcz/tcz.go

```diff
@@ -72,9 +72,10 @@ func clonetree(tree string) error {
 	l.Printf("Clone tree %v", tree)
 	lt := len(tree)
```

Comment:
I don't quite get what this length is for. What's the difference between `path` and `path[lt:]`?